app/views/application/_header.slim

```diff
@@ -0,0 +1,9 @@
+header
+	h1 = link_to "Taskopolis", root_path
+	nav
+		ul
+			- if user_signed_in?
+				li = link_to "Log out", destroy_user_session_path, method: :delete
+			- else
+				li = link_to "Register", new_user_registration_path
+				li = link_to "Log in", new_user_session_path
```

Comment:
Are you using tabs or spaces?  We're going to get all kinds of merge conflicts if we don't use the same thing.

Any objection to moving to 2 spaces?